compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2452,12 +2452,16 @@ TR::Register *OMR::X86::TreeEvaluator::OverflowCHKEvaluator(TR::Node *node, TR::
       //add group
       case TR::ladd:
       case TR::iadd:
+      case TR::iuadd:
```

Comment:
The unsigned math operations are not the strategic direction for OMR. Whether we are checking signed or unsigned overflow is a property of the overflowchk itself and not of the operation underneath it. Please remove iuadd, luadd, buadd from these lists. cadd similar is an suadd and is similarly deprecated.